cf-usb/tasks/csm-generate.sh

```diff
@@ -8,8 +8,8 @@ cd "${SIDECAR_ROOT}"
 if ! test -d vendor ; then
```

Comment:
(Same as above)

Reply:
(See above)

Reply:
Done with commit 345074c906484e2478bf6d1289bd535795c298ff, in testing.